frame/society/src/lib.rs

```diff
@@ -1112,67 +1057,172 @@ decl_error! {
 		/// The caller is not the head.
 		NotHead,
 	}
-}
 
-decl_event! {
-	/// Events for this module.
-	pub enum Event<T, I=DefaultInstance> where
-		AccountId = <T as system::Config>::AccountId,
-		Balance = BalanceOf<T, I>
-	{
-		/// The society is founded by the given identity. \[founder\]
-		Founded(AccountId),
-		/// A membership bid just happened. The given account is the candidate's ID and their offer
-		/// is the second. \[candidate_id, offer\]
-		Bid(AccountId, Balance),
-		/// A membership bid just happened by vouching. The given account is the candidate's ID and
-		/// their offer is the second. The vouching party is the third. \[candidate_id, offer, vouching\]
-		Vouch(AccountId, Balance, AccountId),
-		/// A \[candidate\] was dropped (due to an excess of bids in the system).
-		AutoUnbid(AccountId),
-		/// A \[candidate\] was dropped (by their request).
-		Unbid(AccountId),
-		/// A \[candidate\] was dropped (by request of who vouched for them).
-		Unvouch(AccountId),
-		/// A group of candidates have been inducted. The batch's primary is the first value, the
-		/// batch in full is the second. \[primary, candidates\]
-		Inducted(AccountId, Vec<AccountId>),
-		/// A suspended member has been judged. \[who, judged\]
-		SuspendedMemberJudgement(AccountId, bool),
-		/// A \[candidate\] has been suspended
-		CandidateSuspended(AccountId),
-		/// A \[member\] has been suspended
-		MemberSuspended(AccountId),
-		/// A \[member\] has been challenged
-		Challenged(AccountId),
-		/// A vote has been placed \[candidate, voter, vote\]
-		Vote(AccountId, AccountId, bool),
-		/// A vote has been placed for a defending member \[voter, vote\]
-		DefenderVote(AccountId, bool),
-		/// A new \[max\] member count has been set
-		NewMaxMembers(u32),
-		/// Society is unfounded. \[founder\]
-		Unfounded(AccountId),
-		/// Some funds were deposited into the society account. \[value\]
-		Deposit(Balance),
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config<I>, I: 'static = ()> {
+		#[doc = "Amount of our account balance that is specifically for the next round's bid(s)"]
+		pub pot: BalanceOf<T, I>,
+		pub members: Vec<T::AccountId>,
+		pub max_members: u32,
 	}
+
+	#[cfg(feature = "std")]
+	impl<T: Config<I>, I: 'static> Default for GenesisConfig<T, I> {
+		fn default() -> Self {
+			Self {
+				pot: Default::default(),
+				members: Default::default(),
+				max_members: Default::default(),
+			}
+		}
+	}
+
+	#[pallet::genesis_build]
+	impl<T: Config<I>, I: 'static> GenesisBuild<T, I> for GenesisConfig<T, I> {
+		fn build(&self) {
+			if let Some(founder) = self.members.first().cloned() {
+				<Founder<T, I>>::put(founder.clone());
+				<Head<T, I>>::put(founder);
+			}
+			<Pot<T, I>>::put(&self.pot);
```

Comment:
This also doesnt make sense to me. For the genesis pot to have any value, I expect to see some kind of balance transfer here.